.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,12 +1,42 @@
-<!--
-Thank you for contributing to Snowplow!
+/!\ IMPORTANT /!\
 
-You'll find a small checklist below which should help speed up the review processs:
+As explained in the blog post here https://snowplowanalytics.com/blog/2020/07/16/changing-releasing/ all the components of Snowplow pipeline have been moved to their own repo, PRs should be created directly on the appropriate repo :
 
-- [ ] Have you signed the [contributor license agreement](https://github.com/snowplow/snowplow/wiki/CLA)?
-- [ ] Have you read the [contributing guide](https://github.com/snowplow/snowplow/blob/master/CONTRIBUTING.md)?
-- [ ] Have you added the appropriate unit tests?
-- [ ] Have you run the tests through `sbt test`?
-- [ ] Is your pull request against the `master` branch?
--->
+Trackers
```

Comment:
Do you think it would be better to add a link to the section from `README`? Just to not maintain several duplicated lists.

Reply:
You're right, having the list in different places is a pain